package.json

```diff
@@ -24,27 +24,22 @@
     "@headlessui/react": "^1.7.15",
     "@heroicons/react": "^2.0.18",
     "@vitejs/plugin-react": "^4.0.2",
-    "@yext/sites-components": "1.0.0-rc.4",
-    "classnames": "^2.3.2",
+    "@yext/studio": "^0.25.0",
+    "class-variance-authority": "^0.7.0",
```

Comment:
Can you run `npm i cva@npm:class-variance-authority` and replace all your imports with `import { cva } from "cva";`

Reply:
done

Reply:
sorry can you also remove the `class-variance-authority` import? It repeats right now

Reply:
yep, just did